### DISCUSSION TITLE
how can i use webinterface from mobile?

### DISCUSSION THREAD

**Discussion by neellll** [org-roam/org-roam-ui#325] [Q&A]
how can i access the orui web interface on port 35901 from my local network? i tried from a device connected in the same network 192.168.1. 30:35901 which would be the ip address of my main pc where I have emacs, but the page gives me an error and won't let me access, I thought it was a fw problem but I don't have any restriction locally for this port, so I guess it's something in the configuration , but I'm not good at this, is there actually a way to see my nodes from the webpage from my devices connected under the same network?? currently it turns out that port 35901 is only listening from address 127.0.0.1

**Comment by neellll** [comment]
somehow I managed to partially solve the problem, now from the local network I can reach the orui page but it doesn't let me view any nodes and among the network logs I find the following error

879-09bfd8a9d59d1f41b6f4.js:1 WebSocket connection to 'ws://localhost:35903/' failed: 

I honestly don't understand how I should solve... I have tried everything but for some reason the websocket points to port 35903, is there any way to change the following port or solve the problem?

**Comment by okomestudio** [comment]
This issue appears to be relevant: https://github.com/org-roam/org-roam-ui/issues/293, but that doesn't seem to solve the issue either.

 (I thought this might be just setting 0.0.0.0 as the host, as suggested in the issue ticket, but apparently it isn't.)

Unfortunately, I haven't played around with the code base well enough to offer suggestion as to where in the code to change. 

Hope someone else chimes in.
